assets/inference/environments/mlflow-ubuntu22.04-py312-cpu-inference/context/conda_dependencies.yaml

```diff
@@ -6,4 +6,8 @@ dependencies:
 - python=3.12
 - pip
 - pip:
-  - azureml-inference-server-http=={{latest-pypi-version}}
\ No newline at end of file
+  - azureml-inference-server-http=={{latest-pypi-version}}
+  - azureml-ai-monitoring=={{latest-pypi-version}}
+  - numpy
+  - mlflow
+  - azureml-contrib-services
```

Comment:
Please remove contrib package. Our contrib packages by design do not meet quality bar for public environments.

Reply:
Updated! Removed contrib package.